include/OperatingSystemHelpers.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * OperatingSystemHelpers.h - Helpers for OS related concerns
+ *
+ * Copyright (c) 2025- Michael Gregorius
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#ifndef LMMS_OPERATINGSYSTEMHELPERS_H
+#define LMMS_OPERATINGSYSTEMHELPERS_H
+
+#include "lmmsconfig.h"
+
+#include "qnamespace.h"
+
+
+namespace lmms
+{
+
+constexpr char LADSPA_PATH_SEPERATOR =
+#ifdef LMMS_BUILD_WIN32
+';';
+#else
+':';
+#endif
+
+// Abstract away GUI CTRL key (linux/windows) vs ⌘ (apple)
+constexpr const char* UI_CTRL_KEY =
+#ifdef LMMS_BUILD_APPLE
+"⌘";
+#else
+"Ctrl";
+#endif
+
+constexpr const char* UI_ALT_KEY =
+#ifdef LMMS_BUILD_APPLE
+"Option";
+#else
+"Alt";
+#endif
+
+constexpr const char* getOSSppecificModifierKeyString()
```

Comment:
Any ideas for a better name here?  It's predominantly used for copying patterns and linking automation controls.  I'm fine with something like `copyModifierKeyString`, but I think I will quickly forget the currently proposed name(s).

The edge-case here I think is the "free resizing" -- or in other apps "disable snap-to-grid' which is ironically "Control" in Garageband.  I'm curious what the Windows equivalent is.  From what I've read the "Alt" key is the shortcut that PowerPoint and window snapping uses to disable this behavior.  This suggests that it should be split off into its own in the future, reinforcing the name of `copyModifier` for coping and linking, as "freeResizeModifier" would replace the secondary function.

Reply:
`getCopyModifierString()` sounds like a good name for this function, like tresf says.

Reply:
Sorry for the late replies. I was mainly concentrating on some other open PRs and now I am (too) late to the party. Naming definitively is hard but it seems that a good name has been found. :sweat_smile: